rdrf/rdrf/forms/dsl/parse_utils.py

```diff
@@ -49,16 +56,9 @@ def get_cde_to_section_dict(self):
 CDEInfo = namedtuple('CDEInfo', 'name type allow_multiple is_multi_section formset_prefix')
 
 
+@cache_memoize(settings.CACHE_DEFAULT_TIMEOUT)
```

Comment:
Note:

Looking at how the django database cache works, it will pickle and then base64 encode the data you give it: https://github.com/django/django/blob/f5817c24f44aa90188757ddca1e5e63ba6f5df75/django/core/cache/backends/db.py#L95

Probably a good way to do it, but if we observe it being slow, we could try give it `dict`s rather than proper CDE objects, to make de-serializing quicker.